### PR TITLE
Design/refractor directories

### PR DESCRIPTION
**Pull request by idafum**
Restructured code to use MVVM design pattern and enabled support for data binding.

### CODE REVIEW

File: Models/Card.cs
Comment (VanJackal) [inline Models/Card.cs:1]: this will need to be moved to the wrapper later, fine to stay here though

File: Pages/ProfilePage.xaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+             x:Class="Combat_Critters_2._0.Pages.ProfilePage">
+
+    <!-- <ContentView>
+           <Button
+            Text="Share"
+            Style="{DynamicResource ButtonStyle}"
+            HorizontalOptions="End"
+            VerticalOptions="Center"
+            Clicked="OnShareClicked"/> 
+        </ContentView> -->
```
Comment (VanJackal) [inline Pages/ProfilePage.xaml:13]: dead code

File: Pages/ProfilePage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+using Microsoft.Maui.Controls;
+
+namespace Combat_Critters_2._0.Pages
+{
+    public partial class ProfilePage : ContentPage
+    {
+        public ProfilePage()
+        {
+            InitializeComponent();
+            //BindingContext = new ProfilePageViewModel();
```
Comment (VanJackal) [inline Pages/ProfilePage.xaml.cs:10]: dead code

**Review by VanJackal — CHANGES_REQUESTED** [review]
A few bits of commented out code, otherwise looks good

**Review by idafum — COMMENTED** [review]
Reviewed

File: Models/Card.cs
Comment (idafum) [reply Models/Card.cs:1]: Okay, I wonder how it would be implemented in the wrapper?

**Pull request merged**